vmdb/app/models/ems_kubernetes.rb

```diff
@@ -1,8 +1,4 @@
 class EmsKubernetes < EmsContainer
-  has_many :container_nodes,                      :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
That is part of the patch:

`containers: relationships belong to EmsContainer` https://github.com/simon3z/manageiq/commit/c7f35c86325f1276699c1069a6c699c5822a7b96

and the relationships are just moved to EmsContainer because they're not specific to Kubernetes.